test/components/navbar/user-actions-menu.spec.coffee

```diff
@@ -45,12 +41,12 @@ testWithRole = (roleType) ->
         courseModel.is_concept_coach = false
         CourseActions.loaded(courseModel, courseModel.id)
 
-      it 'should not have disallowed menus', (done) ->
+      it 'should not have disallowed menus', ->
```

Comment:
Why was the `done()` callback removed? Do these tests still run (since they are async & seem to need to call `done()`)?